pkg/applier/manager.go

```diff
@@ -167,6 +170,19 @@ func (m *Manager) createStack(ctx context.Context, stacks map[string]stack, name
 		return
 	}
 
+	log := m.log.WithField("stack", name)
+
+	stackName := filepath.Base(name)
+	if sinceVersion, ignored := m.IgnoredStacks[stackName]; ignored {
```

Comment:
This is a bit misleading as any later versions will not match.

The k0s/version package offers version constraints 🤔

Reply:
Which is the intended behavior. K0s 1.33 should still produce the ignored.txt file referring to k0s 1.32.

Reply:
I'll try to add comments and use better variable names. Otherwise, I'd just remove the version hint.